aten/src/ATen/native/miopen/RNN_miopen.cpp

```diff
@@ -203,7 +203,7 @@ struct TensorDescriptorListParams {
     int64_t batch_sizes_sum;
 
     bool is_input_packed() const {
```

Comment:
@cyyever One more nit: is_input_packed should probably be annotated with [nodiscard] just like empty() is